data-prepper-plugins/kinesis-source/src/test/resources/simple_pipeline_with_extensions.yaml

```diff
@@ -0,0 +1,5 @@
+extensions:
+  kinesis:
+    lease_coordination:
+      table_name: "kinesis-pipeline-kcl"
+      region: "us-east-1"
```

Comment:
How come there is no pipeline config here? Just extension config?

Reply:
Added this as part of the data prepper extensions. This just tests the extensions section for kinesis.